snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/compare/SnomedMapTypeReferenceSetCompareTest.java

```diff
@@ -122,6 +123,23 @@ public void compareDifferentSimpleMapTypeReferenceSets() {
 		assertThat(result.getChangedMembers().values()).anyMatch(m -> TARGET_CODE_3.equals(m.getTargetComponentURI().identifier()));
 	}
 	
+	@Test
+	public void compareLargeSimpleMapTypeReferenceSets() {
+		long startTime = StopWatch.time();
+		final String baseSimpleMapReferenceSet = "900000000000497000";
+		final String  compareSimpleMapReferenceSet = "447562003";
+		ComponentURI baseURI = ComponentURI.of(SnomedTerminologyComponentConstants.SNOMED_SHORT_NAME, SnomedTerminologyComponentConstants.REFSET_NUMBER, baseSimpleMapReferenceSet);
+		ComponentURI compareURI = ComponentURI.of(SnomedTerminologyComponentConstants.SNOMED_SHORT_NAME, SnomedTerminologyComponentConstants.REFSET_NUMBER, compareSimpleMapReferenceSet);
+		
+		CodeSystemRequests.prepareConceptMapCompare(baseURI, compareURI)
+				.build(codeSystemURI)
+				.execute(getBus())
+				.getSync();
```

Comment:
Since you are already using the Java API, which supports timeouts, let's use that for the test case, specify the desired 1 second execution time here with `getSync(1, TimeUnit.SECONDS)`.

Reply:
Done